include/snowflake/client.h

```diff
@@ -210,6 +210,17 @@ typedef enum SF_STATUS {
 #define SF_SQLSTATE_INVALID_CURSOR_POSITION "HY108"
 #define SF_SQLSTATE_OPTIONAL_FEATURE_NOT_IMPLEMENTED "HYC00"
 
+// For Query Context Cache
+#define QCC_CAPACITY_DEF        5
```

Comment:
should we add prefix `SF_` to those contants?

Reply:
fixed